package/yast2-theme.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Mar  2 13:21:43 UTC 2021 - Kenneth Wimer <wimer@suse.com>
+
+- Fixes for bug #1182497
```

Comment:
I think you have to use the bsc#XXXXX format, otherwise our bots will complain:

~~~
- Fixes for bsc#1182497
~~~